web/src/beta/components/Modal/index.tsx

```diff
@@ -67,7 +67,7 @@ const Modal: React.FC<Props> = ({
       isVisible={isVisible}
       title={title}
       onClose={onClose}>
-      <InnerWrapper>
+      <InnerWrapper hasTabs={tabs.length > 0}>
```

Comment:
Instead of this, just add `flex: 1` to ContentWrapper 👍🏻 

Also, can you adjust the margins on the WorkspaceCreationModal's `FormInputWrapper` so the spacing looks more appropriate 🙏🏻

Reply:
done